### DISCUSSION TITLE
在umi3使用了dynamic动态加载组件  umi4中没有找到这个属性  要怎么继续升级umi4

### DISCUSSION THREAD

**Discussion by panyi35** [umijs/umi#8669] [General]
在umi3使用了dynamic动态加载组件  umi4中没有找到这个属性  要怎么继续升级umi4

**Comment by fz6m** [comment]
FYI：

 - https://github.com/umijs/umi/discussions/8254

 - https://github.com/umijs/umi/issues/8578

 - https://umijs.org/docs/guides/directory-structure#loadingtsxjsx
